portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -31,11 +34,21 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String currcomment = getParameter(request, "comment", "");
-    comments.add(currcomment);
-    response.setContentType("application/json;");
-    String json = convertToJson(comments);
-    response.getWriter().println(json);
+    String currcomment = request.getParameter("comment");
```

Comment:
Use camelCame. so currComment instead of currcomment.

Reply:
Done